README.md

```diff
@@ -22,30 +22,61 @@ pip install mkdocs-llmstxt
 Enable the plugin in `mkdocs.yml`:
 
 ```yaml title="mkdocs.yml"
+site_name: My project
+site_description: Description of my project
+site_url: https://myproject.com/  # Required for the llmstxt plugin to work
+
 plugins:
 - llmstxt:
-    files:
-    - output: llms.txt
-      inputs:
-      - file1.md
-      - folder/file2.md
+    markdown_description: Long description of my project
+    sections:
+      Usage documentation:
+        - file1.md
+        - file2.md
+```
+
+The resulting `/llms.txt` file will be available at the root of your documentation.
+With the previous example, it will be accessible at https://myproject.com/llms.txt
+and will contain the following:
```

Comment:
chore: I have stopped wrapping lines manually a few months ago. It produces noisier diffs, but is generally easier to maintain.